python/lsst/galaxy_shear/runShear.py

```diff
@@ -0,0 +1,293 @@
+                os.makedirs(great3_dir)
+                baseArgs = base.split("_")
+                if  baseArgs[0][0] == 'f' and baseArgs[0][1:].isdigit() and len(baseArgs) == 3:
+                    config.filter = int(baseArgs[0][1:])
+                    config.seeing = float(baseArgs[1])
+                    config.shear_value = float(baseArgs[2])
+                    config.psf_dir = os.path.join(config.psf_lib_dir,
+                                        "f%d_%s"%(config.filter, config.seeing))
+                    os.symlink(config.psf_dir, os.path.join(great3_dir, "psfs")) 
+                else:
+                    os.symlink(os.path.abspath(config.psf_dir), os.path.join(great3_dir, "psfs")) 
+                fout = open(os.path.join(great3_dir, "_mapper"), "w")
+                fout.write("lsst.obs.great3.Great3Mapper")
+                fout.close()
+                constants.image_size_deg = config.fov
+                constants.nrows = config.ndims
+                constants.ncols = config.ndims
+                constants.n_subfields =  config.n_subfields
+                constants.xsize["ground"][True] = config.galaxy_stamp_size
+                constants.xsize["ground"][False] = config.galaxy_stamp_size
+                constants.n_subfields_per_field["constant"][True] = 1
+                constants.subfield_grid_subsampling = 1
+                constants.n_deep_subfields = 0
+                constants.deep_frac = 0.0
+                subfield_max = constants.n_subfields + constants.n_deep_subfields - 1
+                if forks > 1:
+                    if len(pidlist) >= forks:
+                        waitforpids(pidlist, waitutil=forks-1)
+                    pid = os.fork()
+                    if pid:
+                        # we are the parent
+                        pidlist.append(pid)
+                    else:
+                        # we are the child
+                        run(base, gal_dir=config.gal_dir, steps=["metaparameters", "catalogs", "config",],
+                            experiments=[config.exp_type], obs_type="ground", shear_type=["constant"], 
+                            draw_psf_src = '%s/control/ground/constant/psfs/psfs.index'%base, 
+                            subfield_max=subfield_max,
+                            shear_value=config.shear_value,
+                            shear_angle=config.shear_angle
+                        )
+                        sys.exit(0)
+                else:
+                    run(base, gal_dir=config.gal_dir, steps=["metaparameters", "catalogs", "config",],
+                        experiments=[config.exp_type], obs_type="ground", shear_type=["constant"], 
+                        draw_psf_src = '%s/control/ground/constant/psfs/psfs.index'%base, 
+                        subfield_max=subfield_max,
+                        shear_value=config.shear_value,
+                        shear_angle=config.shear_angle
+                    )
```

Comment:
I think it might be a bit cleaner to put the implementation for each of these commands in a separate function, so runShear itself just had to test its options and call those functions, with `waitforpids` calls in between.